src/tests/ftest/network/cart_self_test.py

```diff
@@ -78,6 +79,8 @@ def setUp(self):
             self.server_managers[0].get_config_value("provider")
         self.cart_env["OFI_INTERFACE"] = \
             self.server_managers[0].get_config_value("fabric_iface")
+        self.cart_env["DAOS_AGENT_DRPC_DIR"] = "/var/run/daos_agent/"
+
         self.server_managers[0].manager.assign_environment(self.cart_env, True)
 
         # Start the daos server
```

Comment:
same here. we dont need to dump attach info line 89-93

Reply:
https://github.com/daos-stack/daos/commit/d4022c644a8d1f51b48ec3f97ff0ec9cdadee0d8#diff-ca093b7ae7587c88bc15cd3bd5d64269766f939a55c829b79ee6ec6c1d8c02c3L89